src/components/look/look-list/style-tag-list/index.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { message } from 'antd';
+
+import { Button } from '@src/component/atoms';
+
+import { StyleTag } from '@src/types';
+import styleTagsData from '@src/data/style-tags.json';
+import { LookService } from '@src/services';
+
+type StyleTagListProps = {
+  styleTags: StyleTag[];
+  id: number;
+};
+
+function StyleTagList({ styleTags, id }: StyleTagListProps) {
+  const [tags, setTags] = useState<StyleTag[]>([...styleTags]);
```

Comment:
```suggestion
  const [selectedTagIds, setSelectedTagIds] = useState<number[]>(styleTags.map(styleTag => styleTag.id));
```

1. 변수 이름 명확하게 변경
2. 저장해야하는 정보만 저장! (전체가 아닌 id만)

Reply:
오 마자요 이렇게 하고 싶었는데 문법을 몰랐어요ㅠㅠ! 자스 공부좀 해야겠네여ㅎㅎㅎ

Reply:
d3d4728